calico-enterprise/operations/ebpf/troubleshoot-ebpf.mdx

```diff
@@ -45,140 +41,179 @@ If pods or hosts within your cluster have trouble accessing services, check the
+  |          | by policy                      |       0 |     11 | N/A |
+  |          | failed decapsulation           |       0 |      0 | N/A |
+  |          | failed encapsulation           |       0 |      0 | N/A |
+  |          | failed to create conntrack     |       0 |      0 | N/A |
+  |          | fragment of yet incomplete     |       0 |      0 | N/A |
+  |          | packet                         |         |        |     |
+  |          | fragment out of order within   |       0 |      0 | N/A |
+  |          | host                           |         |        |     |
+  |          | fragments not supported        |       0 |      0 | N/A |
+  |          | incorrect checksum             |       0 |      0 | N/A |
+  |          | malformed IP packets           |       0 |      0 | N/A |
+  |          | packets hitting blackhole      |       0 |      0 | N/A |
+  |          | route                          |         |        |     |
+  |          | packets with unknown route     |       0 |      0 | N/A |
+  |          | packets with unknown source    |       0 |      0 | N/A |
+  |          | packets with unsupported IP    |       0 |      0 | N/A |
+  |          | options                        |         |        |     |
+  |          | too short packets              |       0 |      0 | N/A |
+  | Other    | packets hitting NAT source     |       0 |      0 | N/A |
+  |          | collision                      |         |        |     |
+  | Redirect | neigh                          |       0 |      0 | N/A |
+  |          | peer                           |       0 |      0 | N/A |
+  |          | plain                          |      20 |      0 | N/A |
+  | Total    | packets                        |      34 |     22 | N/A |
+  +----------+--------------------------------+---------+--------+-----+
+  ```
 
 ## eBPF program debug logs
 
-$[prodname]'s eBPF programs contain optional detailed debug logging. Although the logs can be very verbose (because
-the programs will log every packet), they can be invaluable to diagnose eBPF program issues. To enable the log, set the
-`bpfLogLevel` Felix configuration setting to `Debug`.
+Sometimes it is necessary to examine the logs that are emitted by the eBPF
+programs themselves.  Although the logs can be very verbose (because the
+programs will log every packet), they can be invaluable to diagnose eBPF program
+issues. To enable the log, set the `bpfLogLevel` Felix configuration setting to
+`Debug`.
 
 :::caution
 
 Enabling logs in this way has a significant impact on eBPF program performance.
 
 :::
 
-> The logs are emitted to the kernel trace buffer, and they can be examined using the following command:
+To reduce the performance impact in production clusters, you can target logging
+to specific traffic and/or specific interfaces using the
+[bpfLogFilters](../../reference/felix/configuration.mdx#BPFLogFilters) Felix
```

Comment:
The path for this file is different for CE and CC. The bad markdown link is what's killing the build this time. Try `/reference/component-resources/node/felix/configuration`.